tfgetent does not compile on FreeBSD

tfgetent is found by configure:

  tfgetent             yes

But compilation fails:

/bin/sh ./libtool  --tag=CC   --mode=compile cc -DHAVE_CONFIG_H -I.   -I/usr/local/include/syslog-ng -I/usr/local/include/glib-2.0 -I/usr/local/include   -I/usr/local/include/eventlog -I./modules/getent   -O2 -pipe -fno-strict-aliasing -MT modules/getent/modules_getent_libtfgetent_la-tfgetent.lo -MD -MP -MF modules/getent/.deps/modules_getent_libtfgetent_la-tfgetent.Tpo -c -o modules/getent/modules_getent_libtfgetent_la-tfgetent.lo `test -f 'modules/getent/tfgetent.c' || echo './'`modules/getent/tfgetent.c
libtool: compile:  cc -DHAVE_CONFIG_H -I. -I/usr/local/include/syslog-ng -I/usr/local/include/glib-2.0 -I/usr/local/include -I/usr/local/include/eventlog -I./modules/getent -O2 -pipe -fno-strict-aliasing -MT modules/getent/modules_getent_libtfgetent_la-tfgetent.lo -MD -MP -MF modules/getent/.deps/modules_getent_libtfgetent_la-tfgetent.Tpo -c modules/getent/tfgetent.c  -fPIC -DPIC -o modules/getent/.libs/modules_getent_libtfgetent_la-tfgetent.o
modules/getent/tfgetent.c:36:10: fatal error: 'features.h' file not found
# include <features.h>

```
     ^
```

1 error generated.
gmake[2]: **\* [modules/getent/modules_getent_libtfgetent_la-tfgetent.lo] Error 1
gmake[2]: Leaving directory `/root/syslog-ng-incubator/work/syslog-ng-incubator-0.3.1'
gmake[1]: *** [all] Error 2
gmake[1]: Leaving directory`/root/syslog-ng-incubator/work/syslog-ng-incubator-0.3.1'
**\* Error code 1

--- a/modules/getent/tfgetent.c
+++ b/modules/getent/tfgetent.c
@@ -33,7 +33,6 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <errno.h>
-#include <features.h>
 #include <stddef.h>
 
 typedef gboolean (*lookup_method)(gchar *key, gchar *member_name, GString *result);

